### PR TITLE
Add community PR info to contribution doc

### PR DESCRIPTION
**Pull request by zivkan**
Fixes: https://github.com/NuGet/Client.Engineering/issues/347

### CODE REVIEW

**Review by zkat — APPROVED** [review]
LGTM :)

**Review by nkolev92 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,11 +18,15 @@
...
+    * Keep the pull request template, and link to an issue [NuGet/Home](https://github.com/NuGet/Home/issues). Create a new issue if you cannot find an existing one. Use a meaningful PR title, not the autogenerated title based on the branch name.
+    * All PRs created by someone outside the NuGet team will be assigned the `Community` label, and a team member will be assigned as the PR shepherd, who will be responsible for making sure the PR gets reviewed (even if they don't review it themselves), and periodically check the PR for progress.
+    * If the NuGet team requests changes and the PR author does not respond within 1 month, a reminder will be added. If no action is taken within 2 months of the reminder, the PR will be closed due to inactivity.
 1. _One-time_: Sign the contributor license agreement, if you haven't signed it before. The [.NET Foundation Bot](https://github.com/dnfclas) will comment on the pull request you just created and guide you on how to sign the CLA.
 1. Submit a doc pull request to the [docs.microsoft-com.nuget](https://github.com/NuGet/docs.microsoft.com-nuget/) repo, if this is a new feature or behavior change.
 
+
```
Comment (nkolev92) [inline CONTRIBUTING.md]: nit: remove extra line.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,8 +18,11 @@
 
 1. [Open an issue here](https://github.com/NuGet/Home/issues) and get some feedback from the NuGet team.
 1. Follow the instructions in [Code](#code)
-1. Make your change, and add tests.
+1. Make your change, and add tests. Please name your branch `dev-<userid>-<very-short-title>`.
 1. Create a [pull request](https://github.com/NuGet/NuGet.Client/pulls).
+    * Keep the pull request template, and link to an issue [NuGet/Home](https://github.com/NuGet/Home/issues). Create a new issue if you cannot find an existing one. Use a meaningful PR title, not the autogenerated title based on the branch name.
```
Comment (donnie-msft) [inline CONTRIBUTING.md]:
```suggestion
    * Create a new issue if you cannot find an existing one [NuGet/Home](https://github.com/NuGet/Home/issues). 
    * Keep the pull request template, and link to an issue. 
    * Use a meaningful PR title, not the auto-generated title based on the branch name.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,8 +18,11 @@
...
+    * All PRs created by someone outside the NuGet team will be assigned the `Community` label, and a team member will be assigned as the PR shepherd, who will be responsible for making sure the PR gets reviewed (even if they don't review it themselves), and periodically check the PR for progress.
```
Comment (donnie-msft) [inline CONTRIBUTING.md:26]:
```suggestion
    * All PRs created by someone outside the NuGet team will be assigned the `Community` label, and a team member will be assigned as the PR shepherd, who will be responsible for making sure the PR gets reviewed (even if they don't review it themselves), and periodically check the PR for progress. Our CI will be run once the shepherd applies the `Approved for CI` label.
```

**Review by donnie-msft — APPROVED** [review]
Made some suggestions

File: CONTRIBUTING.md (hunk 3 above)
Comment (Nirmal4G) [reply CONTRIBUTING.md:26]: Don't forget to add, "To invoke CI again, the label had to be reapplied, every single time!"

File: CONTRIBUTING.md (hunk 2 above)
Comment (zivkan) [reply CONTRIBUTING.md]: The number of community PRs that delete the PR template and then just write whatever they want seems high to me. You don't think there's value in documenting that we want them to keep the template?

Comment (donnie-msft) [reply CONTRIBUTING.md]: See second bullet in my suggestion.

**Comment by nkolev92** [comment]
🔔

**Pull request merged**